static/scss/components/banners.scss

```diff
@@ -86,3 +86,126 @@
         }
     }
 }
+
+// Labels data opt in notification banner
+
+.c-data-notification-banner {
+    display: flex;
+    flex-direction: column;
+    background-color: $color-white;
+    box-shadow: $box-shadow-sm;
+    border-radius: $border-radius-md;
+
+    .data-notification-banner-bg {
+        padding: $spacing-xs;
+        width: 100%;
+        position: relative;
+        min-height: 0;
+    }
+
+    .data-notification-banner-content {
+        padding-left: $spacing-sm;
+        display: flex;
+        flex-direction: row;
+    }
+
+    .data-notification-banner-copy {
+        padding: $spacing-lg;
+    }
+
+    .data-notification-banner-header {
+        font-family: $font-metropolis;
+        font-size: 1.125em;
+        font-weight: 700;
+        text-align: center;
+        width: 90%;
+        padding-bottom: $spacing-sm;
+        margin: 0;
+    }
+
+    .data-notification-banner-header:before {
+        content: "";
+        display: inline-block;
+        height: 16px;
+        width: 16px;
+        margin-right: 12px;
+        background-image: url("/static/images/icon-orange-info.svg");
+        background-size: 16px;
+        background-repeat: no-repeat;
+        background-position: center center;
+    }
+
+    .data-notification-banner-sub {
+        margin: 0;
+        line-height: 150%;
+        padding: $spacing-md 0;
+    }
+
+    .data-notification-banner-link {
+        text-align: center;
+        color: $color-blue-50;
+        margin: 0;
+        font-size: 1rem;
+        font-weight: 600;
+        width: auto;
+        padding: $spacing-lg 0;
+
+        &:hover {
+            color: $color-blue-40;
+        }
+
+        &:focus {
+            color: $color-blue-40;
+            outline: none;
+        }
+    }
+
+    .dismiss-wrapper {
+        width: 100%;
+        height: 0px;
+    }
+
+    .data-notification-dismiss {
+        background: transparent;
+        cursor: pointer;
+        display: block;
+        border: none;
+        float: right;
+        position: relative;
+        z-index: 1;
+        padding: $spacing-md $spacing-md 0 0;
+
+        &:hover {
+            transform: scale(1.05);
+        }
+        
+        img {
+            width: 1.5em;
+        }
+    }
+
+    @media #{$mq-md} {
+
+        .data-notification-banner-copy:before {
+            content: '';
+            position: absolute;
+            z-index: 1;
+            height: calc(100% - #{$spacing-lg});
+            margin-top: -$spacing-lg;
+            width: 5px;
+            margin-left: -$spacing-2xl;
+            background: $color-yellow-50;
```

Comment:
Maybe this colour and the `i` icon could be moved into a separate `.t-warning` class, so that it could be reused for e.g. error messages (with a red line and different icon)?